README.md

```diff
@@ -53,6 +58,10 @@ You can install your host using command:
 Be careful because all data on your hard drive will be erased. You have been
 warned.
 
+If your hard drive has a preexisting swap partition, your live CD might actually be using it, so turn it off:
```

Comment:
@renard once, the disk was already partitioned with a distro installed (former old installation, nothing related to system-install), maybe with a DOS MBR instead of a GPT one. The live CD (I don't remember if Debian or Ubuntu) enabled the swap partition already existing. This prevented the disk to be repartioned (device busy).